udata_gouvfr/templates/nec_mergitur.html

```diff
@@ -0,0 +1,85 @@
+{% extends theme("base.html") %}
+
+{% set meta = {
+    'title': 'Nec Mergitur',
+    'description': "Un Hackathon pour renforcer la sécurité dans Paris",
+    'keywords': [
+        'necmergitur',
+        'paris',
+        'fluctuat',
+        'nec',
+        'mergitur',
+        'hackathon',
+        'évènement',
+    ],
+} %}
+
+{% set body_class="nec_mergitur dataconnexions" %}
```

Comment:
It might be worth renaming the "dataconnexions" class for wider use.
Proposal: contest